securedrop/client.py

```diff
@@ -90,7 +90,12 @@ def register_new_user(self):
 
         pw1 = getpass.getpass(prompt="Enter Password: ")
         pw2 = getpass.getpass(prompt="Re-enter password: ")
+
         if name and email and pw1 and pw2:
+            #enforce password length to min of 12 characters
+            if (len(pw1) < 12) or (len(pw2) < 12):
```

Comment:
You can move this below the auth equality check so only one input's length needs to be checked.

Reply:
This can be removed now